### DISCUSSION TITLE
FundMe.sol cant compile

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#775] [Q&A] [answered]
hey guys. i think that this link @chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol
no longer exists. i cannot compile due to this, which i disregarded in lesson3 but now in lesson 6 the fundme.sol contract is being referenced too again. is anyone else having problems and have suggestions of what i should do. 

to be clear, i have even used patricks source code and this cannot compile either

**Comment by cromewar** [comment]
Hello @mattwilsonbmx well if you look at it, the file actually exists: [click here](https://github.com/smartcontractkit/chainlink-brownie-contracts/tree/main/contracts/src/v0.6/vendor) could you share how's your brownie-config.yaml?

**Reply by ghost** [reply]
before using brownie, literally in remix, when i import @chainlink/contracts/src/v0.6/SafeMathChainlink.sol, and then compile, it refuses to compile saying not found

**Reply by ghost** [reply]
i embedded the code of safemathchainlink.sol in the fund me program and that worked. thanks for the help

**Comment by cromewar** [comment, marked as answer]
Well is a nice Workaround, but I want to give you an explanation:

The file exists as I showed you on the link, but it belongs to `chainlink-brownie-contracts` which is not `chainlink-contracts` they are two different repositories, so Remix is trying to access the second one, and of course is not finding it.
